library/std/src/sys/pal/unix/stack_overflow.rs

```diff
@@ -104,9 +112,11 @@ mod imp {
             rtabort!("stack overflow");
         } else {
             // Unregister ourselves by reverting back to the default behavior.
-            let mut action: sigaction = mem::zeroed();
+            // SAFETY: assuming all platforms define struct sigaction as "zero-initializable"
+            let mut action: sigaction = unsafe { mem::zeroed() };
```

Comment:
note that libc currently defines `struct sigaction` **incorrectly** so don't bother trying to verify whether this is the case!